packages/nimble-components/src/menu-button/tests/menu-button.spec.ts

```diff
@@ -121,6 +121,26 @@ describe('MenuButton', () => {
             expect(element.toggleButton!.disabled).toBeTrue();
         });
 
+        it('should default tabIndex on the toggle button to 0', async () => {
+            await connect();
+            expect(element.toggleButton!.getAttribute('tabindex')).toBeNull();
```

Comment:
The test name doesn't match the assertion here - maybe `should not set tabIndex on the toggle button by default`, or `should not set tabIndex on the toggle button if not set on the menu button`,